dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPI.java

```diff
@@ -77,6 +78,15 @@
    */
   List<ContentType> findAll() throws DotDataException;
 
+  /**
+   * Finds All the Content Types that exists in the system but takes into consideration the {@link LicenseLevel}.
+   * If the {@link LicenseLevel} is Community, only the content types that are not an {@link com.dotcms.contenttype.model.type.EnterpriseType} will be returned
+   *
+   * @return List of Content Types Objects
+   * @throws DotDataException Error occurred when performing the action.
+   */
+  List<ContentType> findAllRespectingLicense() throws DotDataException;
```

Comment:
really we nned a new method here, should not the 'findAll' method respect license?

Reply:
After discussing with @freddyucv we decided it is better to create a new one. So no changes here.

Reply:
After discussing with @freddyucv we decided it is better to create a new one. So no changes here.